experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -2230,10 +2230,73 @@ def _clone_experiment_and_assert_common_expectations(
         return child
 
     def test_get_changelogs(self):
-        experiment = NimbusExperimentFactory.create(slug="experiment-1")
-        generate_nimbus_changelog(experiment, experiment.owner, "Test message")
-        experiment_changelogs = experiment.get_changelogs()
-        self.assertEqual(experiment_changelogs[0].message, "Test message")
+        experiment = NimbusExperimentFactory.create(
+            slug="experiment-1",
+            published_dto={"id": "experiment", "test": False},
+        )
+        user = UserFactory.create()
+        time_format = "%I:%M:%S %p"
+        current_date = datetime.date.today()
```

Comment:
All datetimes that end up in the databse should use `django.utils.timezone` so that they are time-zone aware. datetime.datetime and datetime.date will be created without tzinfo.

If you insert a datetime.datetime into the DB without creating it via `django.utils.timezone` and then get the value out of the db, the values will be different because the one retrieved from the DB will be timezone-aware.

This means that tests that create a date and then check if its equal when it comes out of the database will sometimes fail when running them later in the day.

[docs](https://docs.djangoproject.com/en/3.2/topics/i18n/timezones/)

Reply:
I saw the test cases using datetime so I just followed that, I've updated it to use timezone now :)